pandas/core/series.py

```diff
@@ -1786,7 +1786,10 @@ def count(self, level=None):
 
     def mode(self, dropna=True) -> "Series":
         """
-        Return the mode(s) of the dataset.
+        Return the mode(s) of the series.
```

Comment:
I think "Series" is usually capitalised in the docs - other than that, lgtm, thanks
```suggestion
        Return the mode(s) of the Series.
```